workshops/20_python_beginners_workshop/manual/installation_party.md

```diff
@@ -0,0 +1,152 @@
+1. Download and run the installer. 
+  * When prompted where to install Anaconda, copy the suggested location aside (we'll need it later on) OR change it to something that you will easily find afterwards (like `C:\Anaconda3`).
+
+Once you have your Anaconda installed, there is one trick that will make your life easier. Follow these steps:
+
+6. Press "Windows" key and search for "Edit the system environment variables".
+1. In the "User variables" section (top one) 
+  * press "New...". 
+  * Variable name: `PYTHON3`. 
+  * Variable value: `[Anaconda_dir];[Anaconda_dir]\Scripts;[Anaconda_dir]\Library\bin;` replacing `[Anaconda_dir]` with the directory you copied aside in step 5. 
+  * Press "OK". You just created a new User variable `PYTHON3`. 
+8. Do you have a `Path` User variable?
+  * If not, press "New...". In the window that just showed set Variable name: `Path`. Variable value: `%PYTHON3%`. Press "OK". 
+  * If yes, press "Edit..". In the window that just showed press "New" (that will make the last row of the table editable) and paste `%PYTHON3%` there. 
+  
+In the end it should look something like this:
+![Python Environment Variable](images/environment_variables.png "Python environment variables")
+  
+Now, if everything worked fine you should be able to access Python from any location. Test it:
+1. Press "Windows" key, type "cmd" (that should find a command prompt) and press Enter. 
+2. type `python` and press enter
+3. If everything went well you'll see something like that:
+```
+C:\Users\youruser>python
+Python 3.6.5 |Anaconda, Inc.| (default, Mar 29 2018, 13:32:41) [MSC v.1900 64 bit (AMD64)] on win32
+Type "help", "copyright", "credits" or "license" for more information.
+>>>
+```
+Type `quit()` to exit Python interpreter, and `exit` to shut down the command prompt window.
+
+4. If you see this:
+```
+C:\Users\youruser>python
+'python' is not recognized as an internal or external command,
+operable program or batch file.
+```
+then something went wrong. Try again or search for support on our Slack.
+
+### Linux
+
+Ubuntu (at least as of 16.04 LTE) ships with both Python 3 and Python 2 pre-installed. To make sure our versions are up-to-date run:
+```
+sudo apt-get update
+sudo apt-get upgrade
+```
+typing in your password and agreeing when necessary.
+
+Once the process is complete, we can check the version of Python 3 that is installed in the system by typing:
+
+```
+python3 -V
+```
+
+You will receive output in the terminal window that will let you know the version number. The version number may vary, but it will look similar to this:
+
+```
+Python 3.6.5
+```
+
+To manage software packages for Python, let’s install pip:
+
+```
+sudo apt-get install -y python3-pip
+```
+
+and you're ready to roll! 
+
+Note: pip installs and manages programming packages we may want to use in our development projects. You can install Python packages by typing:
```

Comment:
instead `we` maybe `you` like in the rest of the place. I would personally try to avoid the `you` language, I never got good feedback using it. It can be rephrased to eg. instead of `
You will receive output in the terminal window that will let you know the version number.` --> `The output of this command in the terminal window will show the version number`
Only a suggestion